contentful-uploader/.env.example

```diff
@@ -0,0 +1,2 @@
+MUX_ACCESS_TOKEN_ID=your-access-token-id
```

Comment:
Aren't we replacing these with MUX_TOKEN_ID and MUX_TOKEN_SECRET everywhere?

Reply:
great catch